.changeset/metal-hornets-travel.md

```diff
@@ -0,0 +1,16 @@
+---
+"@fluidframework/container-runtime": patch
+---
+---
+"section": "fix"
+---
+Restored old op processing behavior around batched ops to avoid potential regression
+
+There's a theoretical risk of indeterminate behavior due to a recent change to how batches of ops are processed.
+This fix reverses that change.
+
+Pull Request #21785 updated the ContainerRuntime to hold onto the messages in an incoming batch until they've all arrived, and only then process the set of messages.
```

Comment:
```suggestion
Pull Request #21785 updated the ContainerRuntime to wait to process a set of messages until all messages from an incoming batch have arrived.
```

Reply:
As discussed - Won't Fix (to match 2.3 release notes)